Formula/kube-no-trouble.rb

```diff
@@ -0,0 +1,20 @@
+# Documentation: https://docs.brew.sh/Formula-Cookbook
+#                https://rubydoc.brew.sh/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```

Comment:
As indicated by the audit failure, these will have to be removed.
```suggestion
```

Reply:
will remove it